pkgs/development/python-modules/trytond/default.nix

```diff
@@ -13,44 +12,30 @@
 , werkzeug
 , wrapt
 , passlib
-, pillow
-, bcrypt
 , pydot
 , python-Levenshtein
-, simplejson
 , html2text
-, psycopg2
-, withPostgresql ? true
+, weasyprint
+, gevent
+, pillow
+, withPostgresql ? true, psycopg2
```

Comment:
Please remove the flag. If in one python env both trytonds exists python gets really confused.

Reply:
I think trytond should be removed from pythonPackages anyway. I won't remove it though cause people ought to know they have to use packageOverrides